### DISCUSSION TITLE
Floating UI

### DISCUSSION THREAD

**Discussion by awcodes** [filamentphp/filament#2068] [Ideas]
Hey @danharrin,

As discussed, here's the first pass at an [Alpine plugin for Floating UI](https://github.com/awcodes/alpine-floating-ui).

It's basic at the moment, and probably could use some work, but it's a start.

Maybe @ryangjchandler can put some of his magic on it, since it's more or less a port of his Tooltip plugin. lol.

**Comment by danharrin** [comment]
Hey, this actually looks really good, well done. I wonder if this should be part of the Filament core or if we should just use your package?

**Reply by ryangjchandler** [reply]
@danharrin Are we still planning on moving all of our dropdowns to FloatingUI?

**Reply by danharrin** [reply]
Yeah

**Comment by awcodes** [comment]
Thanks Dan.

I'm perfectly fine with it being in core, but I can see a valid use case for it outside of Filament. Not sure how many people would be interested in it as a standalone plugin though.

I'm good either way.

Been a hectic few weeks. Hopefully I can get back to finishing out some of the other features soon.

Cheers.

**Comment by awcodes** [comment]
Got any tips on how to make the tests work? Floating UI doesn’t export as default and jest errors out. I still have trouble with the 2 JS methodologies.
